integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -191,45 +182,89 @@ <h3 class="heading font-bold text-black">
                             <label>
                                 {% trans "Map" %}
                             </label>
-                            <div id="no_coordinates_alert"
-                                 class="bg-blue-100 border-l-4 border-blue-500 text-blue-500 px-4 py-3 my-2 hidden"
-                                 role="alert">
-                                {% trans "Map preview will be available when coordinates are given." %}
-                            </div>
                             <div id="map"
                                  class="aspect-video"
-                                 data-url="{% url "get_address_from_coordinates" region_slug=request.region.slug %}">
+                                 data-url="{% url "get_address_from_coordinates" region_slug=request.region.slug %}"
+                                 data-bounding-box="{{ request.region.bounding_box.api_representation }}">
```

Comment:
Instead of the center of the region, I'd go with the bounding box when no coordinates are set.